### ISSUE TITLE
Clock divider's output is uneven

### ISSUE THREAD

**Issue by unlessgames** [SVModular/SynthKit#19]
Maybe I'm wrong but shouldn't the 3 output fire on every third pulse.

Here is how it looks like now.

![sk_clock_div_bug](https://user-images.githubusercontent.com/38718816/167118063-f0e1ff7d-94c1-436a-946a-e483f24be411.png)

**Comment by martinkrz** [comment]
I agree and support resolving this issue. In fact, the /3 divider does not uniformly divide time by 3.

There are 4 untriggered beats (7,8,1,2). This is on the github page for the module -- it is how the module works, but I think this is not intuitive.

   1  2  3  4  5  6  7  8
/1  x  x  x  x  x  x  x  x
/2     x     x     x     x
/3        x        x      
/4           x           x
/5              x         
/6                 x      
/7                    x   
/8                       x

You can mitigate this by sending a reset form the 3 division to the module, which will then output a periodic /3 division.

The other dividers (Fibonnaci, prime, etc) also work in this way. For example, the 3 division in the Fibonacci divider triggers only once until 34 counts, which is when the module resets.
